private/buf/buffetch/internal/reader.go

```diff
@@ -232,7 +232,7 @@ func (r *reader) getArchiveBucket(
 		if size < 0 {
 			data, err := io.ReadAll(readCloser)
 			if err != nil {
-				return
+				return nil, err
```

Comment:
```
private/buf/buffetch/internal/reader.go:235:5: naked return in func `getArchiveBucket` with 107 lines of code (nakedret)
                                return
```

This looks to be a valid issue caught by the linter - not sure why it just started reporting it now.